treeherder/webapp/views.py

```diff
@@ -10,3 +11,19 @@ def get_context_data(self, **kwargs):
         # Django_browserid will validate this so it's safe to pass it through.
         context['next'] = self.request.GET.get('next')
         return context
+
+
+class ReftestAnalyzerView(View):
+    """Proxy for the reftest analyzer.
+
+    hg.mozilla.org doesn't serve appropriate Content-Type headers. So proxy
+    the content and add the appropriate header.
+    """
+    ANALYZER_URL = 'https://hg.mozilla.org/mozilla-central/raw-file/tip/layout/tools/reftest/reftest-analyzer.xhtml'
+
+    def get(self):
+        r = requests.get(self.ANALYZER_URL, timeout=5)
+        response = HttpResponse(content=r.data(), status=r.status_code,
```

Comment:
The requests response object doesn't have a method named `data`. You'll either want `text` or `content`:
http://docs.python-requests.org/en/master/user/quickstart/#response-content
https://github.com/kennethreitz/requests/blob/v2.10.0/requests/models.py#L727-L787

We also have a few helper methods that should be used instead. If you're using `.text` then you can use `fetch_text()` in treeherder.etl.common, or else if `.content` then use `make_request()` itself.

Please also place `ANALYZER_URL` in settings.py.